firestore-bundle-server/POSTINSTALL.md

```diff
@@ -0,0 +1,37 @@
+# See it in action
```

Comment:
This file will need some changes. At the very least, it needs to define the format of bundle specifications users put in the appropriate Firestore documents. See go/firestore-bundles-server-repo-doc.

Reply:
I deleted `See it in action` section. Giving a specification is a good idea though.